src/gui/MainWindow.cc

```diff
@@ -1747,6 +1748,43 @@ void MainWindow::actionShowLibraryFolder()
   QDesktopServices::openUrl(QUrl::fromLocalFile(url));
 }
 
+void MainWindow::actionShowBackupFiles()
+{
+  QString dirname (PlatformUtils::backupPath().c_str());
+  QStringList extensions = {};
+  extensions << "scad" << "csg";
+#ifdef ENABLE_PYTHON
+    extensions << "py";
+#endif
+  extensions.replaceInStrings(QRegularExpression("^"), "*.");
+  const auto filter = QString("OpenSCAD Backups (%1)").arg(extensions.join(" "));
```

Comment:
We should not copy whole functions. If it's needed from outside, move `fileOpenFilter()` from anonymous namespace to UIUtils so it's accessible here. Or as alternative extract the getBackupFilename() part and move that to UIUtils.